qubesbuilder/plugins/source_archlinux/__init__.py

```diff
@@ -217,6 +217,7 @@ def run(self):
                     {
                         "packages": packages_list,
                         "source-hash": self.component.get_source_hash(),
+                        "files": ["PKGBUILD"],
```

Comment:
there is also a tarball (listed below as source-archive), shouldn't it be in "files" too?